packages/elements/src/containers/API.tsx

```diff
@@ -69,18 +69,18 @@ const APIImpl: React.FC<APIProps> = props => {
 
   const { data: fetchedDocument, error } = useQuery(
     [apiDescriptionUrl],
-    () => fetch(apiDescriptionUrl).then(res => res.text()),
+    () =>
+      fetch(apiDescriptionUrl).then(res => {
+        if (res.ok) {
+          return res.text();
+        }
+        throw new Error(`Unable to load spec, status code: ${res.status}`);
```

Comment:
Can we say “description” or “document” or “description document” because spec means “OpenAPI itself.” We’ve been consistently moving towards this across the whole ecosystem but spec pops up incorrectly now and then.

Reply:
My bad, fixed